### ISSUE TITLE
Typescript definitions are incorrect for YAxis.showLastLabel

### ISSUE THREAD

**Issue by euangoddard** [highcharts/highcharts#17935]
#### Expected behaviour

This value can be a boolean (as in version 10.2.0). 

#### Actual behaviour

At the moment I get this error:

```
Error: src/stk-charts/configs/stockcharts/security/y-axis.config.ts:85:3 - error TS2322: Type 'boolean' is not assignable to type 'string'.

85   showLastLabel: true,
     ~~~~~~~~~~~~~

  node_modules/highcharts/highcharts.d.ts:170954:5
    170954     showLastLabel?: string;
               ~~~~~~~~~~~~~
    The expected type comes from property 'showLastLabel' which is declared here on type 'YAxisOptions'
```

Previously this correctly supported booleans. I note that the documentation lists the type as `string` for `showLastLabel`, but in the narrative form it says the values can be `true`, `false` or `undefined`. Further, the type for `showFirstLabel` is correctly a boolean.

#### Product version

10.3.1

#### Affected browser(s)

All - this happens at compile time

**Comment by karolkolodziej** [comment]
FYI @bre1470

**Comment by LeahPike** [comment]
I have also encountered this..

![image](https://user-images.githubusercontent.com/6124100/211802229-29c110f7-ef7a-41a2-aa77-09ae93adb49c.png)

**Comment by pawellysy** [comment]
Thank you @LeahPike for your additional input. W'll keep that in mind, when fixing this issue. Thanks!

**Comment by bre1470** [comment]
Hello!
The issue should be fixed with v10.3.3. Can you please check.
Thank you.

**Comment by LeahPike** [comment]
Working for me, thank you :)
